library/lua/time.lua

```diff
@@ -0,0 +1,87 @@
+local _ENV = mkmodule('time')
+
+local TU_PER_DAY = 1200
+--[[
+if advmode then TU_PER_DAY = 86400 ? or only for cur_year_tick?
+advmod_TU / 72 = ticks
+--]]
+local TU_PER_MONTH = TU_PER_DAY * 28
+local TU_PER_YEAR = TU_PER_MONTH * 12
+
+local MONTHS = {
+ 'Granite',
+ 'Slate',
+ 'Felsite',
+ 'Hematite',
+ 'Malachite',
+ 'Galena',
+ 'Limestone',
+ 'Sandstone',
+ 'Timber',
+ 'Moonstone',
+ 'Opal',
+ 'Obsidian',
+}
```

Comment:
This feels like it *should* be a data structure enum (in the df-structures repo and accessed here as `df.months` or something), but I know the code that this came from did it this way too, so I'm not going to force that.

Reply:
Thanks for the tip. Will try that way.

Reply:
@ab9rf what do you think? should this list go in df-structures as an enum?

Reply:
I misread at first, I assumed they were already there. Assumed wrong, as usual. Will wait for you guys to decide.

Reply:
> @ab9rf what do you think? should this list go in df-structures as an enum?

there is no corresponding enum or `#define` in the bay12 info we have received, so if it does it should be scoped to DFHack. toady appears to use an `int32_t` whenever he need to represent a month and uses a magic 12 instead of a `#define` or enum for things like squad schedules